.circleci/test-deploy.yml

```diff
@@ -65,6 +65,13 @@ workflows:
           working-directory: examples/npm-install
           cypress-cache-key: cypress-cache-{{ arch }}-{{ checksum "examples/npm-install/package.json" }}
           post-install: "npm run say-hello && npm run say-goodbye"
+      - cypress/run:
+          filters: *filters
+          name: Custom Node Version Example
+          working-directory: examples/npm-install
+          cypress-cache-key: cypress-cache-{{ arch }}-{{ checksum "examples/npm-install/package.json" }}
+          node-version: "20.15.1"
+          post-install: "npm run say-hello && npm run say-goodbye"
```

Comment:
Should we add something here that actually makes sure it gets the correct node version?
Something like:
```
if ! node --version | grep -q "20.15.1"; then
    echo "Node version 20.15.1 not found"
    exit 1
fi
```

Reply:
I like that, thanks!